packages/server-wallet/src/config/defaults.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-process-env */
+import {
+  LoggingConfiguration,
+  NetworkConfiguration,
+  OptionalDatabaseConfiguration,
+  OptionalServerWalletConfig,
+  ServerWalletConfig,
+} from './types';
+
+export const defaultDatabaseConfiguration: OptionalDatabaseConfiguration & {
```

Comment:
It seems like these defaults are actually the parameters that we expect in our _test environment_. I don't think it's appropriate for such values to act as _default configs_.

Some of them, such as the `host` and `port` of the postgres DB might be reasonable defaults:
- postgres _always_ uses port `5432` by default
- I'm not actually sure about `host`, because I am not sure it is the "default" to run the DB on the same server as the process running the wallet